js/crm.drupal8.js

```diff
@@ -5,7 +5,7 @@ localStorage.setItem('Drupal.toolbar.activeTabID', JSON.stringify('toolbar-item-
 
 (function($) {
   function adjustToggle() {
-    if ($(window).width() < 768) {
+    if ($(window).width() < 768 && $('#toolbar-item-civicrm').length) {
```

Comment:
actually @colemanw won't this potentially lead to https://stackoverflow.com/questions/17583366/javascript-cannot-read-property-length-of-undefined-when-checking-a-variable ?

Reply:
No it won't be a problem. $() returns a jQuery object no matter what.